scripts/start

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env sh
```

Comment:
Because need to run database migrations to fully start the app, this is a handy script so can start app with a single command.  `./scripts/start`.  You can run each command individually if you prefer by following each `docker-compose` line within the script.